### PR TITLE
Escape --reporter=.. to allow for spaces in report path

### PR DESCRIPTION
**Pull request by eskatos**
Follow up to #68 with a better approach, including tests.

### CODE REVIEW

**Comment by eskatos** [comment]
Related to https://github.com/shyiko/ktlint/issues/177

File: plugin/src/test/kotlin/org/jlleitschuh/gradle/ktlint/KtlintPluginTest.kt [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        }
+    }
+
+    private
+    fun withCleanSources() =
+        projectRoot.resolve("src/main/kotlin/source.kt").writeText("""val foo = "bar"""")
+
+    private
+    fun withFailingSources() =
+        projectRoot.resolve("src/main/kotlin/source.kt").writeText("""val  foo    =     "bar"""")
```
Comment (JLLeitschuh) [inline plugin/src/test/kotlin/org/jlleitschuh/gradle/ktlint/KtlintPluginTest.kt]: ❓ Is this formatting (putting `private` on it's own line) in line with the Ktlint formatting?

File: plugin/build.gradle.kts [hunk 2]
```diff
@@ -23,13 +24,16 @@ dependencies {
     compileOnly(kotlin("gradle-plugin", PluginVersions.kotlin))
     compileOnly("com.android.tools.build:gradle:${PluginVersions.androidPlugin}")
     compileOnly("org.jetbrains.kotlin:kotlin-native-gradle-plugin:${PluginVersions.kotlinNativePlugin}")
-    compile("net.swiftzer.semver:semver:${PluginVersions.semver}")
+    implementation("net.swiftzer.semver:semver:${PluginVersions.semver}")
```
Comment (JLLeitschuh) [inline plugin/build.gradle.kts]: 👍

**Comment by Tapchicoma** [comment]
The issue was fixed in ktlint `0.20.0` version.

@eskatos can you rebase your branch against master?

**Comment by JLLeitschuh** [comment]
I think that there should be a warning/error if you try to use a path with spaces and you have the `ktlint` version specified as lower than `2.0`.

**Comment by eskatos** [comment]
I can confirm that using `ktlint-0.20.0` fix the issues.
Closing this PR as invalid

**Pull request closed without merging**